app/views/home/index.html.erb

```diff
@@ -6,12 +6,40 @@
   </p>
 </div>
 
-<h2><%= t('latest_projects') %></h2>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-4">
+      <h2><%= t('latest_projects') %></h2>
 
-<ul>
-  <% cache do %>
-    <% @latest_projects.each do |project| %>
-      <li><%= link_to(project.name, project_path(project.id)) %></li>
-    <% end %>
-  <% end %>
-</ul>
+      <ul>
+        <% cache do %>
```

Comment:
I [looked it up](http://guides.rubyonrails.org/caching_with_rails.html#fragment-caching) and it seems there is no automatic expiration. It's necessary to use a model as the cache key, which will expire based on the `updated_at` field, or manually expire the fragment. Also, to use multiple fragment caches on the same view (which we do) it's necessary to specify a different `action_suffix` parameter for each one.